integrations/ultralytics-yolov5/setup_integration.sh

```diff
@@ -6,5 +6,6 @@
 
 git clone https://github.com/neuralmagic/yolov5.git
 cd yolov5
-git checkout release/0.11
+git checkout release/0.12
 pip install -r requirements.txt
+pip install sparseml[torch, torchvision]
```

Comment:
nit, actually very general observation.
This syntax for `pip install` does not work for mac machines where 
`pip install sparseml'[torch, torchvision]'` is the way to go.
I propose, maybe not now, but in the future, to make have if-else statement (if one command fails ("ubuntu command"), we try to run the alternative("mac command")). 
This would also apply to `setup.py` in SparseML.